### DISCUSSION TITLE
Video is not scrubbing when src contains link to file loaded through SetVirtualHostNameToFolderMapping.

### DISCUSSION THREAD

**Discussion by asv7c2** [MicrosoftEdge/WebView2Feedback#2274] [Q&A]
It is intended behaviour?

**Comment by victorthoang** [comment]
hello @asv7c2 

Are you still looking for an answer to your question? If you still need a response, we can turn this into an Issue with a Question tag (which is typically how we address questions and bugs.

The discussion thread might be more useful for off topic questions or non-issue related inquiries!

Let me know if you'd like me to create an issue from this discussion, or if you'd prefer to add it. Thanks!
